deploy/resources/devel/addons/04-pipelines/deployment-config/buildah/buildah-build-deploy.yaml

```diff
@@ -13,25 +13,40 @@ spec:
   params:
     - name: APP_NAME
       type: string
-  resources:
-    - name: app-source
-      type: git
-    - name: app-image
-      type: image
-
+    - name: GIT_REPO
```

Comment:
Do we want to allow passing the revision of the git repo in addition to the url ?

Reply:
(as it is possible through PR)

Reply:
PR updated